test/functional/eval/input_spec.lua

```diff
@@ -196,6 +228,19 @@ describe('input()', function()
     eq('Vim(call):E118: Too many arguments for function: input',
        exc_exec('call input("prompt> ", "default", "file", "extra")'))
   end)
+  it('supports highlighting', function()
+    feed([[:call input({"highlight": "RainBowParens"})<CR>]])
+    wait()
+    feed('(())')
+    wait()
```

Comment:
@bfredl I got that when I removed all the `wait()`s:

```
./test/functional/ui/screen.lua:302: Row 4 did not match.
Expected:
  |{EOB:~                        }|
  |{EOB:~                        }|
  |{EOB:~                        }|
  |*                         |
  |{RBP1:(}{RBP2:()}{RBP1:)}^                     |
Actual:
  |{EOB:~                        }|
  |{EOB:~                        }|
  |{EOB:~                        }|
  |*{RBP1:(}{RBP2:()}{RBP1:)}^                     |
  |                         |

To print the expect() call that would assert the current screen state, use
screen:snapshot_util(). In case of non-deterministic failures, use
screen:redraw_debug() to show all intermediate screen states.

stack traceback:
        ./test/functional/ui/screen.lua:302: in function 'wait'
        ./test/functional/ui/screen.lua:216: in function 'expect'
        test/functional/eval/input_spec.lua:233: in function <test/functional/eval/input_spec.lua:230>
```

I do not know why though. And I do not see any problems in TUI.